rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
```

Comment:
- [ ] rental should add itself to the @book achieving the 'belongs-to' association, something like so:
```
    book.rentals << self
```

Reply:
Done Thanks for suggestion.